package.json

```diff
@@ -8,9 +8,12 @@
   },
   "typings": "fastify.d.ts",
   "scripts": {
-    "test": "standard | snazzy && tap -j4 test/*.test.js test/*/*.test.js && tsc test/types/index.ts --target es6 --module commonjs --noEmit",
-    "coverage": "tap -j4 --cov --coverage-report=html test/*.test.js test/*/*.test.js",
-    "coveralls": "tap -j4 test/*test.js  test/*/*.test.js --cov",
+    "lint": "standard | snazzy",
+    "unit": "tap -j4 test/*.test.js test/*/*.test.js",
+    "typescript": "tsc test/types/index.ts --target es6 --module commonjs --noEmit",
+    "test": "npm run lint && npm run unit && npm run typescript",
+    "coverage": "npm run unit -- --cov --coverage-report=html",
+    "coveralls": "npm run unit --cov",
     "benchmark": "node ./examples/$npm_config_usingfile & pid=$! && autocannon -c 100 -d 5 -p 10 localhost:3000/ && kill -9 $pid"
```

Comment:
This does not work for me on windows :(

Reply:
Because it is a script for unix.
You can do it in the "old way", with two terminals :)

Reply:
I'm little bit surprised: I've only substitute the scripts. The "&&" and "|"  there were there yet! And I follow the [npm doc](https://docs.npmjs.com/cli/run-script#description)

Anyway, are there some chance to do that under windows?